mkosi/__init__.py

```diff
@@ -3661,6 +3662,16 @@ def make_generated_root(args: MkosiArgs, root: Path, for_cache: bool) -> Optiona
     return None
 
 
+def read_chunked(blob: BinaryIO) -> bytes:
```

Comment:
```suggestion
def read_chunked(blob: BinaryIO) -> Iterator[bytes]:
```

Reply:
That makes more sense. It's no longer required though.